File: Android/app/src/main/java/com/semc/aqi/model/ForecastItem.java
package com.semc.aqi.model;

public class ForecastItem {
    private String Time;
    private String AqiLevel;
    private String Aqi;
    private String PrimaryParameter;
    private String Weather;
    private String Temperature;

    public String getTime() {
        return Time;
    }

    public void setTime(String time) {
        Time = time;
    }

    public String getAqiLevel() {
        return AqiLevel;
    }

    public void setAqiLevel(String aqiLevel) {
        AqiLevel = aqiLevel;
    }

    public String getAqi() {
        return Aqi;
    }

    public void setAqi(String aqi) {
        Aqi = aqi;
    }

    public String getPrimaryParameter() {
        return PrimaryParameter;
    }

    public void setPrimaryParameter(String primaryParameter) {
        PrimaryParameter = primaryParameter;
    }

    public String getWeather() {
        return Weather;
    }

    public void setWeather(String weather) {
        Weather = weather;
    }

    public String getTemperature() {
        return Temperature;
    }

    public void setTemperature(String temperature) {
        Temperature = temperature;
    }
}

File: Server/Eden.Data.Domain/Data/ForecastItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Eden.Domain.Data
{
    public class ForecastItem
    {
        public string Time { get; set; }

        public string AqiLevel { get; set; }

        public string Aqi { get; set; }

        public string PrimaryParameter { get; set; }

        public string Weather { get; set; }

        public string Temperature { get; set; }

    }
}
